modules/azure/function_app_linux_managed_identity/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~> 3.48"
+      version = "~> 3.47"
```

Comment:
Why this change? The result should be the same, it will use the latest provider version within 3.x